PostfixCalculator1/PostfixCalculator.c

```diff
@@ -0,0 +1,94 @@
+#include <stdio.h>
+#include <stdbool.h>
+#include "stack.h"
+#include "postfixCalculator.h"
+
+typedef struct StackElement {
+    int value;
+    struct StackElement* next;
+} StackElement;
+
+typedef struct Stack {
+    StackElement* head;
+} Stack;
+
+bool isOperation(char element) {
+    return element == '+' || element == '-' || element == '*' || element == '/';
+}
+
+bool isDigit(char element) {
+    return element == '0' || element == '1' || element == '2' || element == '3' || element == '4' || element == '5' || element == '6' || element == '7' || element == '8' || element == '9';
```

Comment:
```suggestion
    return element >= '0' && element <= '9';
```
А ещё есть библиотечная функция "isdigit"